README.md

```diff
@@ -832,30 +832,25 @@ cmd
 
 ### Unix
 
-Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD
+Exiv2 can be built on many Unix and Linux distros.  We actively support the Unix Distributions NetBSD and FreeBSD.
 
-#### NetBSD
-
-NetBSD uses pkgsrc as the package manager.  You can build exiv2 with the commands:
+I am willing to support Exiv2 on commercial Unix distributions such as Solaris, AIX, HP-UX and OSF/1 provided you provide with an ssh account on your platform.  I will require super-user privileges to install software.
 
-```bash
-$ cd /usr/pkgsrc/graphics/exiv2
-$ make install
-```
+#### NetBSD
 
-You can build exiv2 from source using the methods described for linux.  I build and installed exiv2 using "Pure CMake" and didn't require conan.
-You will want to use `pkgsrc` to build/install
+You can build exiv2 from source using the methods described for linux.  I built and installed exiv2 using "Pure CMake" and didn't require conan.
```

Comment:
I just realised that we have two styles of writing in the README. Most of the document is written in a impersonal way or giving suggestions from the Exiv2 contributors to the Exiv2 users/clients. In here, however, there  is a personal way of writing to tell personal experiences. I think it would be better to keep consistency in that sense.

Reply:
Let's leave it for now.  I'm quite careful to review/revise the docs for every dot.  It's better to avoid the first person in docs.  If I don't fix this for v0.27.2, I will fix it in v0.27.3 (and of course 0.28).